### PR TITLE
Switch to log.trace where appropriate

### PR DESCRIPTION
**Pull request by lalexgap**
Related to #2057.

Switchs multiple log.info to log.trace so logging output will be reduced.

Currently the `LOG_LEVEL` is left on `trace` for production but that can easily be flipped when we want to deploy a quieter version.

### CODE REVIEW

File: packages/web3torrent/src/clients/payment-channel-client.ts [hunk 1]
```diff
@@ -452,7 +452,7 @@ export class PaymentChannelClient {
       );
     } catch (e) {
       if (e.message === 'User declined') {
-        log.info('User declined budget creation');
+        log.trace('User declined budget creation');
```
Comment (andrewgordstewart) [inline packages/web3torrent/src/clients/payment-channel-client.ts:455]: I think it's worth taking some time to think about what should be `info`, `trace`, etc. 

For instance, this is catching an error, and it doesn't happen repeatedly. So I don't think it should be `trace`.

A good rule of thumb seems to be:
A. things that happen repeatedly should tend have log level equal to `debug` or `trace`
B. things we might want to know about, to see what's going on, should have log level equal to `info`, `warn` or `error`

Did you apply some kind of rule of thumb like this?

**Comment by andrewgordstewart** [comment]
I changed the comment, because I don't think we want to close #2057 even with these changes -- we shouldn't have ~100MB log files at the trace level when downloading a ~100MB file

File: packages/web3torrent/src/clients/payment-channel-client.ts (hunk 1 above)
Comment (lalexgap) [reply packages/web3torrent/src/clients/payment-channel-client.ts:455]: While this is catching an "error" technically, it's essentially logging out that the user has declined the budget creation by clicking a button.

In my mind this is useful information to have if we're trying to debug what went wrong in some situation. However there doesn't seem to be a huge benefit to logging this out in the normal process, as we're essentially echoing the action the user took in the UI.

My rough rule of thumb is that `info` is used for messages that might be of interest for advanced users and provide insight into what is going on. `trace` should be used for messages that help us trace what was going on when something went wrong. (Although I only did a quick pass through `web3torrent` so the log level might not be right for all of them.

File: .env.production-ropsten [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ## all
 NODE_ENV = 'production'
 LOG_DESTINATION = 'console'
-
+LOG_LEVEL= 'trace' # TODO: This should be set to info before we release
```
Comment (andrewgordstewart) [inline .env.production-ropsten:4]: Let's create an issue to track this

**Pull request merged**